regparser/tree/xml_parser/reg_text.py

```diff
@@ -73,13 +73,27 @@ def build_tree(reg_xml):
 
     part = reg_xml.xpath('//PART')[0]
 
+    # Build a list of SUBPARTs, then pull SUBJGRPs into that list:
     subpart_xmls = [c for c in part.getchildren() if c.tag == 'SUBPART']
+    subpart_and_subjgrp_xmls = []
     if len(subpart_xmls) > 0:
-        subparts = [build_subpart(reg_part, s) for s in subpart_xmls]
-        tree.children = subparts
+        for subpart in subpart_xmls:
+            subpart_and_subjgrp_xmls.append(subpart)
+            subjgrps = [c for c in subpart.getchildren() if c.tag == 'SUBJGRP']
+            for subjgrp in subjgrps:
+                subpart_and_subjgrp_xmls.append(subjgrp)
```

Comment:
I tend to flip-flop on where exactly I fall between concise and explicit; I would probably favor the latter option of `subjpart_and_subjgrps.extend(subjgrps)` because that's still quite clear about what's going on. I'll make that change.

As for SUBJGRPs existing outside SUBPARTs, yes, I should probably try addressing that; I've only been testing against 479. I'm not sure it should hold up this merge, though.